portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -110,6 +110,13 @@ definition {
 			locator1 = "FormFields#COLOR_SWATCH_ITEM");
 	}
 
+	macro editConfirmationFieldText {
+		Type(
+			key_fieldName = "${fieldName}",
+			locator1 = "FormFields#TEXT_CONFIRMATION_FIELD",
+			value1 = "${fieldValue}");
```

Comment:
@brunofariasdeo, should we check if the element exists before starting type on it? Does the macro `Type` already has it internally? How about invoke `FormFields.viewConfirmationFieldTextLabel` here?

If we don't guarantee that the element is present before try to type on it, then we can introduce flakiness on this test, what do you think?

Reply:
Hi Renato,

I thought that once we already had an `AssertElementPresent` on `viewConfirmationFieldTextLabel`, it would not be required. But I believe that it would be add another layer to this macro and make it 'flaky-proof'. I have changed it accordingly.